guide/popular-topics/webhooks.md

```diff
@@ -62,14 +62,29 @@ Once you are there, click on the `Create Webhook` button on the top right. This
 
 ### Creating webhooks with discord.js
 
-Discord.js provides a method for creating webhooks called <branch version="11.x" inline>[`TextChannel#createWebhook()`](https://discord.js.org/#/docs/main/11.5.1/class/TextChannel?scrollTo=createWebhook)</branch><branch version="12.x" inline>[`TextChannel#createWebhook()`](https://discord.js.org/#/docs/main/stable/class/TextChannel?scrollTo=createWebhook)</branch>.
+<branch version="11.x">
+
+Discord.js provides a method for creating webhooks called <branch version="11.x" inline>[`TextChannel#createWebhook()`](https://discord.js.org/#/docs/main/11.5.1/class/TextChannel?scrollTo=createWebhook)</branch>.
```

Comment:
The previous approach (with inline branch tags for the docs link) duplicates less content 
only the codeblock needs to be wrapped in additional branch blocks

Reply:
Amended